src/test/java/DukeTest.java

```diff
@@ -0,0 +1,10 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
According to the module's coding standards, should `import static org.junit.jupiter.api.Assertions.assertEquals;` be put above `import org.junit.jupiter.api.Test;`? The same comment for all other classes that have those two imports.